firmware/hw_layer/stepper_dual_hbridge.cpp

```diff
@@ -36,19 +50,57 @@ bool DualHBridgeStepper::step(bool positive) {
 		return false;
 	}
 
+	if (CONFIG(stepperNumMicroSteps) > 1) {
+		float dutyMult = CONFIG(stepperMaxDutyCycle) * phaseDutyCycleDivisor;
+		int numStepIncr = maxNumSteps / CONFIG(stepperNumMicroSteps);
+		if (!positive)
+			numStepIncr = -numStepIncr;
+		for (int i = CONFIG(stepperNumMicroSteps); i > 0; i--) {
+			m_phase = (m_phase + numStepIncr) & tableSizeMask;
+			update(dutyMult);
+			// sleep 1/Nth of the pause time
+			pause(CONFIG(stepperNumMicroSteps));
+		}
+		return true;
+	}
+
     // step phase, wrapping
     if (positive) {
         m_phase = (m_phase + 1) & 0x03;
     } else {
         m_phase = (m_phase - 1) & 0x03;
     }
 
+    update(1.0f);
```

Comment:
should non-micro mode use `dutyMult`?

Reply:
Maybe. 'dutyMult' is 0 by default (not 100) and will break the default behavior for existing users. Some measures have to be made...